pgx/src/dune

```diff
@@ -1,6 +1,17 @@
+(* -*- tuareg -*- *)
+
+let preprocess =
+  match Sys.getenv "BISECT_ENABLE" with
+  | "yes" -> "(preprocess (pps bisect_ppx ppx_jane))"
+  | _ -> "(preprocess (pps ppx_jane))"
+  | exception Not_found -> "(preprocess (pps ppx_jane))"
```

Comment:
What changed here?

Reply:
This will allow publishing pgx to opam without needing to remove bisect_ppx from opam files or dune files. Bisect is a dev only dependency, but without this we'd be forced to mark it as a test dependency. This will avoid having any end-user needing to install bisect just to use pgx

Reply:
i've moved this back to the dune file from `master`. We can decide on how to define bisect_ppx as a dependency at a later stage.